File: test/data/usecase/src/cs/UpdateLeafFlag.cs
if (phase >= 1 && phase< 4)
{
    if (LeafNumber > 0)
    {
        if (HasFlagLeafLiguleAppeared == 0 && (FinalLeafNumber > 0 && LeafNumber >= FinalLeafNumber))
        {
            HasFlagLeafLiguleAppeared = 1;
            if  (calendarMoments.Contains("FlagLeafLiguleJustVisible")==false)
            {
                calendarMoments.add("FlagLeafLiguleJustVisible");
                calendarCumuls.add(cumulTT);
                calendarDates.add(currentdate);
            }
        }
    }
    else:
        HasFlagLeafLiguleAppeared = 0;
}


    

File: test/data/usecase/src/py/UpdateLeafFlag.py
if (phase >= 1 and phase< 4):
    if (LeafNumber > 0):
        if (HasFlagLeafLiguleAppeared == 0 and (FinalLeafNumber > 0 and LeafNumber >= FinalLeafNumber)):
            HasFlagLeafLiguleAppeared = 1
            if  "FlagLeafLiguleJustVisible" not in calendarMoments:
                calendarMoments.append("FlagLeafLiguleJustVisible")
                calendarCumuls.append(cumulTT)
                calendarDates.append(currentdate)


    else:
        HasFlagLeafLiguleAppeared = 0


    
